cc-proxy.service.in

```diff
@@ -3,7 +3,7 @@ Description=Clear Containers Proxy
 Documentation=https://github.com/clearcontainers/proxy
 
 [Service]
-ExecStart=@libexecdir@/cc-proxy
+ExecStart=@libexecdir@/clearcontainers/cc-proxy
```

Comment:
can we standardise on clear-containers? see https://github.com/clearcontainers/runtime/issues/85